resume-buddy/src/main/webapp/style.css

```diff
@@ -1,6 +1,6 @@
-body {
+* {
```

Comment:
I think the reason you are seeing the margin issue when `body` is here is that margin is not a property that passes down to children, unlike `font-family`.

Is the goal to actually have every single element on the page centered? If so, I think the `margin` should go into it's own style block and things that are specifically for `body`, like the `max-width` should stay in this block for `body`.

Reply:
I probably am not understanding CSS / HTML in this case. So the margin gets picked up by everything you need when using `*` or when adding a class to `body`, but not when the CSS is on the `body` tag itself?